Trying to run "make", receives this error.

Any idea? :3  

alon @ ~/kvm-test/kvm-kernel-example - [master] $ **sudo make**
make -C hypervisor
make[1]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/hypervisor'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/hypervisor'
make -C kernel
make[1]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/kernel'
make -C syscalls/ syscalls.a
make[2]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/kernel/syscalls'
make[2]: 'syscalls.a' is up to date.
make[2]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/kernel/syscalls'
make -C hypercalls/ hypercalls.a
make[2]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/kernel/hypercalls'
make[2]: 'hypercalls.a' is up to date.
make[2]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/kernel/hypercalls'
make -C mm/ mm.a
make[2]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/kernel/mm'
make[2]: 'mm.a' is up to date.
make[2]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/kernel/mm'
make -C elf/ elf.a
make[2]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/kernel/elf'
make[2]: 'elf.a' is up to date.
make[2]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/kernel/elf'
make -C utils/ utils.a
make[2]: Entering directory '/home/alon/kvm-test/kvm-kernel-example/kernel/utils'
cc -nostdlib -Os -Wall -Werror -fPIE -pie -masm=intel -I.. -c -MMD -MP string.c
string.c: In function ‘strlen’:
string.c:6:10: error: ‘strnlen’ specified bound 18446744073709551615 exceeds maximum object size 9223372036854775807 [-Werror=stringop-overflow=]
    6 |   return strnlen(s, ~0ll);
      |          ^~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [../template.mk:16: string.o] Error 1
make[2]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/kernel/utils'
make[1]: *** [Makefile:24: utils/utils.a] Error 2
make[1]: Leaving directory '/home/alon/kvm-test/kvm-kernel-example/kernel'
make: *** [Makefile:6: kernel] Error 2

--- a/kernel/utils/string.c
+++ b/kernel/utils/string.c
@@ -3,7 +3,7 @@
 /* no memory accessibility checks in these string functions */
 
 uint64_t strlen(const char *s) {
-  return strnlen(s, ~0ll);
+  return strnlen(s, (1ull << 63) - 1);
 }
 
 uint64_t strnlen(const char *s, uint64_t maxlen) {

